From 6736d29307f7f0359f771f2b070e7946c7fe894c Mon Sep 17 00:00:00 2001
From: CanYang He <hcy@rock-chips.com>
Date: Tue, 26 Mar 2019 10:35:26 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3399: VDD_CENTER need fix to 0.9V

some frequency not fix VDD_CENTER to 0.9V, although these frequency not
enable, to prevent these frequency being enable, fix all VDD_CENTER to
0.9V

Change-Id: I711bf091962f1ca1508b2611bfcaf26e37e451fd
Signed-off-by: CanYang He <hcy@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-lp4.dts | 4 ++--
 arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts            | 4 ++--
 arch/arm64/boot/dts/rockchip/rk3399-tve1030g.dtsi            | 4 ++--
 3 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-lp4.dts b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-lp4.dts
index a3c564346511..581e80a970fd 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-lp4.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-lp4.dts
@@ -139,12 +139,12 @@
 
 	opp-200000000 {
 		opp-hz = /bits/ 64 <200000000>;
-		opp-microvolt = <825000>;
+		opp-microvolt = <900000>;
 		status = "disabled";
 	};
 	opp-300000000 {
 		opp-hz = /bits/ 64 <300000000>;
-		opp-microvolt = <850000>;
+		opp-microvolt = <900000>;
 		status = "disabled";
 	};
 	opp-400000000 {
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts b/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts
index 0b389d9c1c4c..700dec8e8d19 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts
@@ -284,12 +284,12 @@
 
 	opp-200000000 {
 		opp-hz = /bits/ 64 <200000000>;
-		opp-microvolt = <825000>;
+		opp-microvolt = <900000>;
 		status = "disabled";
 	};
 	opp-300000000 {
 		opp-hz = /bits/ 64 <300000000>;
-		opp-microvolt = <850000>;
+		opp-microvolt = <900000>;
 		status = "disabled";
 	};
 	opp-400000000 {
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-tve1030g.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-tve1030g.dtsi
index 665c40d886b6..fb7dc0455374 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-tve1030g.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-tve1030g.dtsi
@@ -235,12 +235,12 @@
 
 	opp-200000000 {
 		opp-hz = /bits/ 64 <200000000>;
-		opp-microvolt = <825000>;
+		opp-microvolt = <900000>;
 		status = "disabled";
 	};
 	opp-300000000 {
 		opp-hz = /bits/ 64 <300000000>;
-		opp-microvolt = <850000>;
+		opp-microvolt = <900000>;
 		status = "disabled";
 	};
 	opp-400000000 {
-- 
2.35.3

